package.json

```diff
@@ -2,6 +2,18 @@
   "name": "mocha",
   "version": "7.2.0",
   "description": "simple, flexible, fun test framework",
+  "main": "./index.js",
+  "exports": {
+    ".": [
+      {
+        "require": "./index.js",
```

Comment:
```suggestion
```

Providing the `require` condition will cause an experimental warning for `Conditional exports` to be emitted from node.js 13.7.0, 13.8.0 and 13.9.0 (when conditional exports were unflagged but still warned).  This warning would be emitted not just for `import mocha from 'mocha'` but also for `require('mocha')`.  Dropping the `require` condition will cause `require('mocha')` to use the `default` condition which never had a warning.

Sorry I didn't get a chance to post here earlier, I wasn't on a PC.

Reply:
Oh, wow. This is really bad. Thanks for the heads up! And I thought I was pretty up to date on ESM and conditional exports, and here I am missing stuff that would have _broken Mocha_ (and I consider such a warning as breaking Mocha). But why would removing the `require` section and keeping the `default` one _not_ show a warning? A fluke that somehow benefits us, or was it intentional?

Reply:
"Conditional exports" is a feature which expands on "exports".  On it's own "exports" gives the ability to remap any specifier (not just main).  So like main can say what `require('mocha')` resolves to, `"exports": {"./tdd": "./lib/tdd.js", "./": "./"}` would say that `require('mocha/tdd')` would actually load `node_modules/mocha/lib/tdd.js`.  On it's own "exports" didn't differentiate between import and require.  "default" was actually part of the original "exports" feature, thus no warning.

Support for "require" and "import" conditions were added as part of conditional exports (possibly "node" as well but I don't remember for sure on that one).  Declaring the "import" condition in package.json isn't enough to trigger the warning but `import('mocha')` on node.js 13.7.0 - 13.9.0 will warn about conditional exports being an experimental feature.  With this latest change `require('mocha')` will will fallback to "default" thus no warning.